### PR TITLE
Fix prediction error in transfer learning.

### PR DESCRIPTION
**Pull request by cxxgtxy**

### CODE REVIEW

File: beginner_source/transfer_learning_tutorial.py [hunk 1]
```diff
@@ -253,7 +253,7 @@ def visualize_model(model, num_images=6):
             images_so_far += 1
             ax = plt.subplot(num_images//2, 2, images_so_far)
             ax.axis('off')
-            ax.set_title('predicted: {}'.format(dset_classes[labels.data[j]]))
+            ax.set_title('predicted: {}, truth {}'.format(dset_classes[preds.numpy()[j][0]], dset_classes[labels.data[j]]))
```
Comment (chsasank) [inline beginner_source/transfer_learning_tutorial.py]: Thanks, 
But, there's not enough space to show both truth and predicted.
Can you please use:
`ax.set_title('predicted: {}'.format(dset_classes[preds[j,0]]))`

**Comment by cxxgtxy** [comment]
Anyone check it?

File: beginner_source/transfer_learning_tutorial.py [hunk 2]
```diff
@@ -253,7 +253,7 @@ def visualize_model(model, num_images=6):
             images_so_far += 1
             ax = plt.subplot(num_images//2, 2, images_so_far)
             ax.axis('off')
-            ax.set_title('predicted: {}'.format(dset_classes[labels.data[j]]))
+            ax.set_title('predicted: {}'.format(dset_classes[preds.numpy()[j][0]]))
```
Comment (chsasank) [inline beginner_source/transfer_learning_tutorial.py]: `preds[j, 0]` should do fine, no?

**Comment by chsasank** [comment]
Thanks for reminding. Was a bit busy. Just added a comment, please review it and I'll merge once fixed.

File: beginner_source/transfer_learning_tutorial.py (hunk 2 above)
Comment (cxxgtxy) [inline beginner_source/transfer_learning_tutorial.py]: You are right.It works.

**Comment by chsasank** [comment]
Thanks!

**Pull request merged**